features/speech-recognition.yml

```diff
@@ -3,6 +3,8 @@ description: The `SpeechRecognition` API converts audio into text using the devi
 spec: https://webaudio.github.io/web-speech-api/#speechreco-section
 group: speech
 caniuse: speech-recognition
+status:
+  compute_from: api.SpeechRecognition.SpeechRecognition
```

Comment:
Seems like we should have the option of `compute_excluding` when we want to exclude a smaller fraction of the set of keys.